htop.c

```diff
@@ -281,6 +342,12 @@ int main(int argc, char** argv) {
 
    CommandLineSettings flags = parseArguments(argc, argv); // may exit()
 
+#ifdef HAVE_LIBCAP
+   if (flags.dropCapabilities && getuid() == 0 && dropCapabilities() < 0) {
```

Comment:
Why limit this to root only?

Reply:
Cause unprivileged users probably don't have CAP_SETCAP.

Reply:
If I read this correctly you don't need to:
https://bugzilla.kernel.org/show_bug.cgi?id=4604#c3

Reply:
dropped the `getuid()` condition